industry_benchmarks/utils/results_cleanup.py

```diff
@@ -56,6 +56,62 @@ def compute_mbar_energies(analyzer):
     return u_ln, N_l
 
 
+def save_mbar_energies(
```

Comment:
I got the same DG values when using the data from the old_energies.npz compared to gather.

Reply:
:tada: 

That is great! @IAlibay do you think this is ready to ship now?

Reply:
Yeah I checked the size, this adds 5.3 MB extra per edge, but I can live with that